.github/workflows/unit-tests-refresh.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: sudo containers/unit-tests/start CC=clang distcheck
 
       - name: Run i386 gcc distcheck test
-        run: sudo containers/unit-tests/start :i386 NO_NPM=1 distcheck
+        run: sudo containers/unit-tests/start :i386 distcheck
```

Comment:
Shouldn't this use `WEBPACK_PACKAGES=''` then, to also explicitly test that  mode?